lis/plugins/LIS_lsmda_pluginMod.F90

```diff
@@ -153,10 +153,11 @@ module LIS_lsmda_pluginMod
 ! !INTERFACE:
 subroutine LIS_lsmda_plugin
 !EOP
-
+! Hans Lievens added DA_PF
 #if ( ( defined DA_DIRECT_INSERTION ) || \
       ( defined DA_ENKS )             || \
-      ( defined DA_ENKF ) )
+      ( defined DA_ENKF )             || \
+      ( defined DA_PF ) )
```

Comment:
Is the DA_PF working? 
Is it a requirement for this pull request about S1 Snow Depth Assimilation? 
If not, it would be better to keep this out here ... just wondering.

Reply:
I don't think the PF is working in its current form... so yes keeping it out sounds good